DrewBradley/rancid-tomatillos
# Feature/search bar function

### What does this PR do?
- This PR adds functionality to our search input in the header component. A user can now search movies by movie title.

### What does it fix?
- No bugs to fix.

### Is this a feature or a fix?
- This is a feature, allowing a user to filter movies by movie title.

### Where should the reviewer start?
- Clone this repo down to your machine. Run npm install then run npm start. 

### How should this be tested?
- Go to the search bar in the top right of the Grouch Potato app. Try typing in a movie title or any letters and see the main display change, only rendering movies that include the characters in the input.
